program/check-palindrome-string/CheckPalindromeString.rb

```diff
@@ -0,0 +1,12 @@
+puts "Please enter a string: "
+input_string = gets.chomp
+
+def checkPalindromeString string
+    if string == string.reverse
+        "Palindrome String"
```

Comment:
@jfinley6 can you please convert both the strings to lowercase before comparison , so that this will work for case sensitive strings as well